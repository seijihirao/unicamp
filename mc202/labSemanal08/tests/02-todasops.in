inserir 6122
inserir 3937
inserir 8924
inserir 2553
inserir 5031
inserir 5897
inserir 7613
inserir 9452
inserir 1381
inserir 4410
em-ordem
pre-ordem
pos-ordem
largura
excluir 5897
excluir 1381
excluir 9452
largura
terminar
