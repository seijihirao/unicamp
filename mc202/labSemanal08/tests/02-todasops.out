1381 2553 3937 4410 5031 5897 6122 7613 8924 9452 
6122 3937 2553 1381 5031 4410 5897 8924 7613 9452 
1381 2553 4410 5897 5031 3937 7613 9452 8924 6122 
6122 3937 8924 2553 5031 7613 9452 1381 4410 5897 
6122 3937 8924 2553 5031 7613 4410 
